libs/platform/src/lib/dynamic-page/dynamic-page.component.ts

```diff
@@ -102,6 +102,10 @@ export class DynamicPageComponent extends BaseComponent implements AfterContentI
     @Input()
     expandContent = true;
 
+    /** Heading level of the dynamic page title. */
+    @Input()
+    headingLevel: number;
```

Comment:
You need some type safety for this. The heading level can be a number from 1 to 6. The user should be notified if the number in not one of these.